### PR TITLE
Update review process guidelines, fix minor typos

### PR DESCRIPTION
**Pull request by llewelld**
## Summary

Updates the review process guidelines to explain that reviews are
available, but need to be requested.

Plus some very small fix-ups to address typos.

## Related issues
N/A

## Screenshots
N/A

---

### Updates

Updated title, summary and commit message to better reflect the actual changes.

### CODE REVIEW

**Comment by llewelld** [comment]
Just some really minor things I noticed while reading through.

**Comment by llewelld** [comment]
Rebased.

File: content/docs/contributing/contributing_changes.md
Comment (JimMadge) [inline content/docs/contributing/contributing_changes.md:165]: There are already words to this effect on the page. It would be good to avoid repeating ourselves.

Do you think it is better placed here than above?
Or, we could make that warning box a shortcode and reuse it in multiple places.

Comment (llewelld) [reply content/docs/contributing/contributing_changes.md:165]: Can I ask what the intended policy is here? I totally agree it's better to avoid the repetition, and I think the choice of how it's presented may effect how it's understood.

When I read the existing page last week the red box at the top implied (to me, at least) that it would be better if I *didn't* ask for reviews and should just auto-merge. Whereas the text at the end gave me the impression that I really *ought* to ask for reviews, even if they're not being enforced.

Obviously this isn't my call, but personally I'd much rather have my changes reviewed. In that case I'd drop the red box and leave the text at the bottom unchanged.

Comment (JimMadge) [reply content/docs/contributing/contributing_changes.md:165]: You can and should!

There is some discussion of this [here](https://github.com/alan-turing-institute/REG-handbook/discussions/10) which you can contribute to. **At the end of the day. The handbook should be owned by the team so how it works is whatever the team decides**.

Originally, I thought it would take quite a bit of effort to promote the handbook and make contributing/updating the book a natural thing for all team members to do (I think this is still true). So I thought it would be good to have some 'handbook people'. The idea then was to put the burden on the handbook managers/reviewers. Contributors submit the content and then reviewers can worry about linting/rendering/style _etc._.

(I was also influenced by Sarah, a former team member, who made a strong case that reviews are more welcoming/inclusive because it removes the expectation for contributors to be experts and able to solve all problems themselves)

I think we then found that not introducing any barriers was important for adoption, so we decided to not require reviews (only a subset of CI tests passing). This hasn't changed since, but also the rules as explained in the book haven't been updated.

Comment (llewelld) [reply content/docs/contributing/contributing_changes.md]: If I'm understanding correctly, the current policy is that users are allowed to either merge without review, or wait for a review at their discretion. I've updated the text to try to align with this.

I removed the red warning from the top and moved it all to the "Review Process" section at bottom of the page. My reasoning is that the warning message made me nervous (because it's presented as a warning), and also I feel the info is better kept all together in one place. However, I can change this around if you think otherwise?

Comment (JimMadge) [reply content/docs/contributing/contributing_changes.md]: That's right. In fact, pointing out that you can and should ask for a review if you want one is important.

Moving it makes sense. I think my idea was to put it right at the begining, but actually I think it fits better where it has been moved to.

Maybe the red danger box was too strong? I wanted to have it in an admonition box as it contradicted the rest of the page. However, that is not the case now so I don't think it is necessary.

File: content/docs/contributing/contributing_changes.md
Comment (JimMadge) [inline content/docs/contributing/contributing_changes.md:163]:
```suggestion
However, if you'd like your changes reviewed you can suggest reviewers when, or after, opening your pull request. 
If a reviewer then requests any changes or makes any comments, these should be resolved before the pull request is merged.
```

At the moment, there is no maintainer group and no one is automatically prompted to review (but we can of course change this if we have maintainers).

**Review by JimMadge — CHANGES_REQUESTED** [review]

File: content/docs/contributing/contributing_changes.md
Comment (llewelld) [reply content/docs/contributing/contributing_changes.md]: Thanks for the suggested change; I've incorporated it in.

**Comment by llewelld** [comment]
I've updated the commit message to better reflect the changes going on here.

**Review by JimMadge — APPROVED** [review]
LGTM :+1:

I think this is much better as the process has been this way for a while and doesn't seem likely to change in the near future.

**Comment by JimMadge** [comment]
@all-contributors please add @llewelld for documentation

**Pull request merged**
